File: app/src/main/java/com/cliff/conch/box/scene/IActivityTaskManager.java
package com.cliff.conch.box.scene;

import com.cliff.conch.box.reflect.RefClass;
import com.cliff.conch.box.reflect.RefStaticMethod;

public class IActivityTaskManager {
    public static Class<?> TYPE = RefClass.load(IActivityTaskManager.class, "android.app.IActivityTaskManager");

    public static class Stub {
        public static Class<?> TYPE = RefClass.load(Stub.class, "android.app.IActivityTaskManager$Stub");
        public static RefStaticMethod<?> asInterface;
    }
}

File: reflect-android/src/main/java/reflect/android/app/IActivityTaskManager.kt
package reflect.android.app

import com.cliff.reflection.common.IReflect
import com.cliff.reflection.common.annotation.PStaticMethod
import com.cliff.reflection.common.annotation.ProxyClass

@ProxyClass("android.app.IActivityTaskManager")
interface IActivityTaskManager : IReflect {
    companion object

    @ProxyClass("android.app.IActivityTaskManager\$Stub")
    interface Stub : IReflect {
        companion object

        @PStaticMethod("com.cliff.wrapper.service.IActivityTaskManager")
        fun asInterface(obj: android.os.IBinder): Any
    }
}